src/CMakeLists.txt

```diff
@@ -25,6 +26,7 @@ set(SRC
     stdlib_experimental_error.f90
     stdlib_experimental_kinds.f90
     stdlib_experimental_optval.f90
+    stdlib_experimental_stats.f90
```

Comment:
I don't believe this entry should be here, since `stdlib_experimental_stats.f90` is generated by `fypp`.